validator/client/attest_protect.go

```diff
@@ -19,9 +19,9 @@ func (v *validator) preAttSignValidations(ctx context.Context, indexedAtt *ethpb
 	fmtKey := fmt.Sprintf("%#x", pubKey[:])
 	if featureconfig.Get().LocalProtection {
 		v.attesterHistoryByPubKeyLock.RLock()
-		attesterHistory := v.attesterHistoryByPubKey[pubKey]
+		attesterHistory, ok := v.attesterHistoryByPubKey[pubKey]
 		v.attesterHistoryByPubKeyLock.RUnlock()
-		if isNewAttSlashable(attesterHistory, indexedAtt.Data.Source.Epoch, indexedAtt.Data.Target.Epoch) {
+		if ok && isNewAttSlashable(attesterHistory, indexedAtt.Data.Source.Epoch, indexedAtt.Data.Target.Epoch) {
```

Comment:
Is this way of handling the nil data too silent? Should something be logged?

Reply:
maybe at a debug level. It is a bit concerning that it is nil